Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased (in master)
 
+- Added subtree view to file listings.
```

Comment:
Changed to 'recursive' in the messaging. Also updated the Changelog and added more detail.